plugins/scheduled_task/services/ScheduledTaskProfileService.php

```diff
@@ -290,4 +290,57 @@ protected function createScheduledTaskJob(ScheduledTaskProfile $scheduledTaskPro
 
 		return $batchJob;
 	}
+	
+	/**
+	 * @action getExclusiveTask
+	 * @param int $runnerType
+	 * @param int $maxTime
+	 * @return KalturaScheduledTaskProfile
+	 */
+	public function getExclusiveTaskAction($runnerType, $maxTime)
+	{
+		if ($runnerType)
```

Comment:
i think it makes more sende to wirtue it as:
if (!$runnerType)
  $runnerType = '*'
$profile = kScheduledProfileTaskAllocator::allocateObjectByTag(kScheduledProfileTaskAllocator::OBJECT_NAME, $runnerType, $maxTime)